app/org/sagebionetworks/bridge/play/controllers/SurveyController.java

```diff
@@ -51,7 +56,8 @@ public Result getAllSurveysMostRecentVersion() throws Exception {
         return okResult(surveys);
     }
     
-    public Result getAllSurveysMostRecentVersion2() throws Exception {
+    // This is "2V3" because Play Framework cannot route two URLs to the same code.
+    public Result getAllSurveysMostRecentVersion2V3() throws Exception {
```

Comment:
2V3 is identical to V3. Since we're refactoring stuff anyway, can we have 2V3 just call through to V3 instead of duplicating the code?